alarm-frontend/src/Components/Math.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { BASE_URL } from '../constants';
+import { pauseAlarm } from "../sound";
+
+
+
+function Math({ difficulty, setCurrentType }) {
+    const [num1, setNum1] = useState();
+    const [num2, setNum2] = useState();
+    const [length, setLength] = useState(difficulty);
+    const [formData, setFormData] = useState('');
+    const [correct, setCorrect] = useState();
+    
+    if (length === 0) {
+        pauseAlarm();
+        setCurrentType("Base");
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (formData === correct) {
+            setLength(length-1);
+        } else {
+            alert("Incorrect!")
+        }
+    }
+
+    useEffect(function askAlarmAPI() {
+        async function getMathProblem() {
+            const response = await axios.get(`${BASE_URL}/Math`);
+            setNum1(response.data[0].first);
+            setNum2(response.data[0].second);
+            setCorrect(response.data[0].first + response.data[0].second);
```

Comment:
Why do we need to key into the first element? Can your endpoint return the object instead of a list of objects?